src/richie/apps/search/indexers/courses.py

```diff
@@ -1,6 +1,8 @@
 """
 ElasticSearch course document management utilities
 """
+# pylint: disable=consider-using-f-string
```

Comment:
why disable it so largely? It's better to use f-string everywhere we can so leave this rule on the file?

Reply:
I don't know exactly why but current pylint rule to disable the warning `consider-using-f-string` was ignored since the upgrade. 

I finally find a nicer fix : Add the rule `# pylint: disable-next=consider-using-f-string` line 296.